test/runtests.jl

```diff
@@ -2,6 +2,7 @@ using PGFPlotsX
 using Base.Test
 using Compat
 using DataStructures: OrderedDict
+using DataFrames
```

Comment:
We should make sure to `using` all deps here because otherwise we get recompilation messages in the generated documentation.

Reply:
OK, will do.